.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of your changes in the Title above -->
```

Comment:
Maybe I was just looking to the future! 😄 What about just the stuff about the checkmarks ?